### PR TITLE
Revealjs & update scorecard action version

### PR DESCRIPTION
**Pull request by izar**
Added a template to create a https://reveal.js presentation.

**Needs tweaks**!

### CODE REVIEW

**Comment by colesmj** [comment]
Just a note - reveal.js is at https://revealjs.com/

File: .github/workflows/scorecard.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@ jobs:
           persist-credentials: false
 
       - name: "Run analysis"
-        uses: ossf/scorecard-action@99c53751e09b9529366343771cc321ec74e9bd3d # v2.0.6
+        uses: ossf/scorecard-action@v2.3.1
```
Comment (colesmj) [inline .github/workflows/scorecard.yml:40]: Is this change related to the revealjs addition?

File: docs/reveal.md
Comment (colesmj) [inline docs/reveal.md:1]: Do you have a sample of what this produces (maybe with the default threat model)?

Comment (izar) [reply docs/reveal.md:1]: I'll add the movie to the readme - I don't want to pollute the tree with spurious files.

File: .github/workflows/scorecard.yml (hunk 1 above)
Comment (izar) [reply .github/workflows/scorecard.yml:40]: Let me remove this, it should be on an update of its own.

**Review by colesmj — APPROVED** [review]

**Comment by colesmj** [comment]
@izar Did you plan to remove / revert the change to the ossf scorecard? If so I'll hold off on the merge so you can revert that change.

**Comment by izar** [comment]
Nope, it's ok to go forward as is. Must have been some inconsistency between my local repo, most probably some mistake by me in git ops, but 2.3.1 is the right one.
